src/Popover/elements.js

```diff
@@ -1,34 +1,113 @@
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
-export const PopOver = styled.div`
-position: relative;
+export const ARROW_SIZE = 8;
+export const CONTENT_PADDING = 1.8;
+export const MARGIN = 12;
 
-width: ${({ width }) => width};
+export const Arrow = styled.span`
+  color: ${({ theme: { palette } }) => palette.white};
 
-padding: 1.8rem;
+  text-shadow: ${({ placement }) =>
+    placement.startsWith('bottom')
+      ? '1px -4px 6px hsla(0,0%,0%,0.1)'
+      : placement.startsWith('left')
+      ? '1px -5px 5px hsla(0, 0%, 0%, 0.1)'
+      : placement.startsWith('right')
+      ? '1px -5px 5px hsla(0, 0%, 0%, 0.1)'
+      : '1px -5px 5px hsla(0, 0%, 0%, 0.1)'};
 
-border-radius: ${({ theme: { dimensions } }) => dimensions.radius};
+  position: absolute;
+  display: inline-block;
+  font-size: ${ARROW_SIZE * 2}px;
 
-box-shadow: 0 0.2rem 1.6rem 0 hsla(0, 0%, 0%, 0.1), 0 0.2rem 1.6rem 0 hsla(206, 23%, 69%, 0.1),
-  0 0 0 0.1rem hsl(207, 22%, 90%);
+  transform: ${({ placement }) =>
```

Comment:
It's a bit redundant, move this part of code directly the right place, where you check for placement and add css